src/Dapi/Products/DapiRequest.cs

```diff
@@ -13,7 +13,7 @@ public class DapiRequest {
             NullValueHandling = NullValueHandling.Ignore
         };
 
-        private static readonly IRestClient httpClient = new RestClient()
+        private static readonly IRestClient httpClient = new RestClient(new RestClientOptions { Timeout = 300000 })
```

Comment:
The `RestClientOptions` is introduced in `RestSharp` v107, and we are using v106. So you have to find another way, otherwise we will have to migrate to the new version and make sure there are no major changes required from our side.

Reply:
ffbb7e5cc2556ebeaaa98787100ffa7214aa716a